src/core/Akka/Dispatch/AbstractDispatcher.cs

```diff
@@ -242,13 +323,13 @@ protected MessageDispatcher(MessageDispatcherConfigurator configurator)
         /// <summary>
         /// The ID for this dispatcher.
         /// </summary>
-        public string Id { get; internal set; }
+        public string Id { get; protected set; }
```

Comment:
No need to set these values externally anymore - handled by the `Dispatcher` class.